basil/TL/Serial.py

```diff
@@ -40,9 +40,9 @@ def close(self):
 
     def write(self, data):
         if self.write_termination is None:
-            self._port.write(bytes(data))
+            self._port.write(bytes(data, 'utf-8'))
```

Comment:
Why is this needed?

Reply:
the constructor `bytes` apparently needs this explicit argument now with python 3. `'utf-8'` should have been the default, so it was always implicitly used